src/main/resources/interface-specification.yml

```diff
@@ -0,0 +1,398 @@
+openapi: 3.0.3
+info:
+  title: attribute registry Process Micro Service
+  description: Offers operations to retrieve attribute registry data
+  version: '{{version}}'
+  contact:
+    name: API Support
+    url: 'http://www.example.com/support'
+    email: support@example.com
+  termsOfService: 'http://swagger.io/terms/'
+  x-api-id: an x-api-id
+  x-summary: an x-summary
+servers:
+  - url: '/attribute-registry-process/{{version}}'
+    description: This service is the attribute registry process
+security:
+  - bearerAuth: []
+tags:
+  - name: attribute-registry
+    description: Implements attribute-registry process
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+  - name: health
+    description: Verify service status
+    externalDocs:
+      description: Find out more
+      url: 'http://swagger.io'
+paths:
+  '/attributes':
+    parameters:
+      - $ref: '#/components/parameters/CorrelationIdHeader'
+      - $ref: '#/components/parameters/IpAddress'
+    post:
+      summary: Creates attribute
+      tags:
+        - attribute
+      operationId: createAttribute
+      responses:
+        '201':
+          description: Attribute created
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Attribute'
+        '400':
+          description: Bad Request
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '409':
+          description: Conflict
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+      description: Creates the attribute passed as payload
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/AttributeSeed'
+        description: Contains the attribute data to save.
+    get:
+      summary: 'returns the list of attributes available on the registry'
+      tags:
+        - attribute
+      operationId: getAttributes
+      parameters:
+        - in: query
+          name: search
+          schema:
+            type: string
+      responses:
+        '200':
+          description: array of currently available attributes
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/AttributesResponse'
+        '404':
+          description: Attributes not found
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+      description: Returns the list of currently available attributes
+  '/bulk/attributes':
```

Comment:
@beetlecrunch probably we can ditch the `bulk` specific endpoint and just consider the list of ids as a generic parameter of the `/attributes` endpoint, because it will retrieve data from read model

Reply:
> @beetlecrunch probably we can ditch the `bulk` specific endpoint and just consider the list of ids as a generic parameter of the `/attributes` endpoint, because it will retrieve data from read model

In this case, the `search` parameter of the `/attributes` will be used as the list of ids (Such as the `ids` parameter of the `bulk` endpoint)? Or should we add another parameter for that?